tests/steps/gdc_TARGET_case2_harmonized.yaml

```diff
@@ -69,8 +69,7 @@ ResearchSubject:
         # renamed; Kathy says that this should be null for the sample:
         derived_from_specimen: Initial sample
         File:
-          - associated_project:
-            - TARGET-ALL-P2
+          - associated_project: TARGET-ALL-P2
```

Comment:
@druth314 is this something we need for R1 or can be done later?

Reply:
I think we can wait. Files records that belong to more than one project will show up more than once, and that info is not entirely lost.

Reply:
@pshapiro4broad if you have time, could you leave these as lists. Otherwise, I can do it.

Reply:
I would say that this is related to our plan to aggregate/merge the data more widely, e.g. by specimen/file. It's fine if we leave it as a single value and address this after R1.

Reply:
I can revert this. But to make this work, I'd need to also change the transform code so it produces lists.

Reply:
@pshapiro4broad please leave this as a list since both @druth314  and @YeastCell are ok with leaving it as a list (this was what was originally in the code, this PR changes it to a string).